apps/script/HoC2023AiGenerateWeights.py

```diff
@@ -1,158 +1,162 @@
-# Script to generate the associated output weights contained in files like cached-spacy-background-map.json that are used to calculate final effects output HoC2023
+# Script to generate the associated output weights contained in files like cached-spacy-background-map.json
+# that are used to calculate final effects output HoC2023
+# Run this Python script from the code-dot-org root directory.
 import spacy
 import json
 
-# Load the spaCy model
-nlp = spacy.load("en_core_web_md")
+# Load the spaCy natural language processing model - English tokenizer, tagger, parser and NER (named entity recognition)
+nlp = spacy.load("en_core_web_lg")
 
-# Define your "id" (emojis) and foreground/background/palette lists
-# For items like backgrounds/foregrounds, certain values were adjusted to reflect the most context-rich but open-ended word within the phrase
-# e.g. "ripples_random" is adjusted to "ripples", "smiling_poop" -> "poop", etc.
-# This was done due to spacy interpreting underscored phrases as a unique word and phrases such as "smiling poop" not correlating highly with most anything within a internet scraped corpus
-id = ["poopy", "romantic", "party", "silly", "sparkle", "happy", "magic", "spooky", "cute", "funky", "wavy", "lights", "rainbow", "robot", "chaotic", "disco", "zen", "fast", "evil", "cold", "cosmic", "sad", "black-and-white", "warm", "cool"]
-palettes = ["cool", "electronic", "ice cream", "neon", 'rave', "tropical","vintage",'warm', 'greyscale', 'sky', 'ocean', 'sunrise', 'sunset', 'spring', 'summer', 'autumn', 'winter', 'twinkling', 'rainbow', 'roses']
-backgrounds = ['circles',
-    'color',
-    'diamonds',
-    'disco',
-    'fireworks',
-    'swirl',
-    'kaleidoscope',
-    'lasers',
-    'splatter',
-    'rainbow',
-    'snowflakes',
-    'text',
-    'galaxy',
-    'sparkles',
-    'spiral',
-    'disco',
-    'stars',
-    'music',
-    'ripples',
-    'random',
-    'quads',
-    'flowers',
-    'squiggles',
-    'stars',
-    'petals',
-    'clouds',
-    'grid',
-    'starburst',
-  ]
+ai_inputs_file = open('apps/static/dance/ai/ai-inputs.json')
+data = json.load(ai_inputs_file)
```

Comment:
nit: something like `ai_inputs` or `emoji_inputs` or `emoji_json` instead of just `data`?

Reply:
If you don't mind - I'll add to the follow-up. Would like to try to get this merged soon. Thanks!

Reply:
https://github.com/code-dot-org/code-dot-org/pull/54527